pkg/apis/build/v1alpha1/buildrun_types.go

```diff
@@ -272,6 +272,11 @@ func (br *BuildRun) IsCanceled() bool {
 	return br.Spec.State != nil && *br.Spec.State == BuildRunStateCancel
 }
 
+// GetLabels returns the labels of the BuildRun
+func (br *BuildRun) GetLabels() map[string]string {
+	return br.Labels
+}
```

Comment:
I don't think this is needed because Kubernetes already provides that function on `ObjectMeta`.